Tests/Grid/Mapping/SourceTest.php

```diff
@@ -0,0 +1,208 @@
+<?php
+
+namespace APY\DataGridBundle\Grid\Tests\Mapping;
+
+use APY\DataGridBundle\Grid\Mapping\Source;
+use PHPUnit\Framework\TestCase;
+
+class SourceTest extends TestCase
+{
+    public function setUp()
+    {
+        $this->source = new Source([]);
+    }
+
+    public function testSetterColumnsWithDefaultValue()
+    {
+        $this->assertAttributeEquals(
+            [],
+            'columns',
+            $this->source
+        );
+    }
+
+    public function testSetterFilterableWithDefaultValue()
+    {
+        $this->assertAttributeEquals(
+            true,
+            'filterable',
+            $this->source
+        );
+    }
+
+    public function testSetterSortableWithDefaultValue()
+    {
+        $this->assertAttributeEquals(
+            true,
+            'sortable',
+            $this->source
+        );
+    }
+
+    public function testSetterGroupsWithDefaultValue()
+    {
+        $expectedGroups = array('0' => 'default');
+
+        $this->assertAttributeEquals(
+            $expectedGroups,
+            'groups',
+            $this->source
+        );
+    }
+
+    public function testSetterGroupByWithDefaultValue()
+    {
+        $this->assertAttributeEquals(
+            [],
+            'groupBy',
+            $this->source
+        );
+    }
+
+    public function testSetterColumns()
```

Comment:
Even here, you're not testing a setter but the constructor, so, `testConstructWithColumn` is a better name to me (it's valid for all test below)